### PR TITLE
change browser dir to lib and add PostMessageTransportRemote.js

### PR DESCRIPTION
**Pull request by AJDBenner**

### CODE REVIEW

File: lib/PostMessageTransportRemote.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        setCallbacks: function (callbacks) {
+            this._callbacks = callbacks;
+        },
+        
+        /**
+         * Sends a message over the transport.
+         * @param {string} msgStr The message to send.
+         */
+        send: function (msgStr) {
+            parent.window.postMessage(JSON.stringify({
```
Comment (humphd) [inline lib/PostMessageTransportRemote.js]: This should just be `parent.postMessage`

File: lib/PostMessageTransportRemote.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            // Listen for message.
+            window.addEventListener("message", function (event) {
+                console.log("[Brackets LiveDev] Got message: " + event.data);
+                if (self._callbacks && self._callbacks.message) {
+                    self._callbacks.message(event.data);
+                }
+            });
+        }
+    };
+    global._Brackets_LiveDev_Transport = postMessageTransport;
```
Comment (humphd) [inline lib/PostMessageTransportRemote.js:51]: Add a newline above this line.

File: lib/iframe-browser.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+define(function (require, exports, module) {
```
Comment (humphd) [inline lib/iframe-browser.js:1]: You needed to use `git mv` so it sees this as a rename vs. a new file.  Can you fix?

**Comment by humphd** [comment]
You're going to need to also change `main.js` because it's going to be looking in the wrong directory for the `iframe-browser.js` file now that you've renamed it to `lib/`

**Comment by humphd** [comment]
I think this is r+.  @sedge, can you do a final look and OK it too?

**Comment by sedge** [comment]
@AJDBenner You need to do a rebase cuz we've merged in code since you started this work.

**Pull request merged**